src/Components/Search/SearchBar.tsx

```diff
@@ -357,8 +381,9 @@ export const SearchBarRefetchContainer = createRefetchContainer(
           term: { type: "String!", defaultValue: "" }
           hasTerm: { type: "Boolean!", defaultValue: false }
         ) {
-        search(query: $term, mode: AUTOSUGGEST, first: 5)
+        search(query: $term, mode: AUTOSUGGEST, first: 7)
           @include(if: $hasTerm) {
+          totalCount
```

Comment:
Technically this isn't the correct count to use. This returns the total count for an auto suggest style query, whereas the search results page does keyword search, and so the actual count you'll see when you click on this item and go to the search results page is likely to be different.

You can check out how we're getting the count for search results here: https://github.com/artsy/reaction/blob/0aba9cc92bc99464bf1e4209301bd78eba7f39be/src/Apps/Search/SearchApp.tsx#L35-L45

Not sure how much of a blocker it is to get this number to exactly match what you'd see on search results!

Reply:
Great callout @mzikherman, thanks! This requirement came from @cavvia, so I'm tagging him here and would love some direction about what was envisioned here. I just reached for the closest thing, but yeah, would probably be best to have this number match the number they see when they go to the results page!

Reply:
I'm assuming that was the intent and it's probably safe to go ahead and implement that. I was just calling out why `totalCount` as-is wouldn't get you there.

Reply:
@mzikherman reading through this again this morning with fresh eyes, I think this is what we're discussing here:

The total count being used in this PR is wrong - what I've got here is the total number of suggestions. Were the user to follow the first item to the results page, the totals wouldn't match. In order for these to line up, I'd have to adjust my query like so:

```graphql
{
  suggestionSearch:search(query: $term, mode: AUTOSUGGEST, first: 7) {
    ...
  }
  totalCountSearch:search(query: $term, first: 1, aggregations: [TYPE]) {
    ...
  }
  filter_artworks(keyword: $term, size: 0, aggregations: [TOTAL]) {
    ...
  }
}
```

The `suggestionSearch` gives us the results we need to draw the suggestions, the `totalCountSearch` gives us the aggregations for non-artwork results and the `filter_artworks` gives us the results for artworks. I could then use the code snippet you quoted above to compute the same total from the results page.

Did I get this right? Is this what we want to do? Are there performance implications here that I'm not seeing? This seems like an expensive operation to send to MP, but maybe I'm missing something!

Reply:
That is definitely correct, and that is definitely expensive!!

It's up to design (cc @cavvia ) to decide if the performance hit (both to the user and to ES which is going to see 3 simultaneous queries for every new autosuggest fetch), if it's worth it or not. I don't think performance is _necessarily_ a blocker (this is only on interactions with search, and not on page load)...but it's worth bringing up.

I was wondering how much of a blocker it is to get this number to exactly match what you'd see on the search results page. Because if it is...it's 2 additional ES calls _just_ for that count (on top of the autosuggest results).

Reply:
Yes, this is expensive - we should only consider this if we can guarantee that fetching the counts will not block the rendering of the auto-suggest results in any way, those need to load as quickly as possible... it's crucial that this interaction be very snappy.

Reply:
Ok, just talked this through with @jeffreytr and we brainstormed some ways to move forward here:

1. drop the count from the first item in suggestions
2. use the total count from suggestions even though it could potentially not
   match results page
3. use the expensive operation and hope for the best in terms of performance
4. delay the rendering of the count so that suggestions render first, then we
   fetch the count

We agreed that options 2 and 3 are not viable. That left us with a choice between dropping the count altogether for this AB test or exploring another approach of splitting up the queries to not block the rendering of the suggestions.

In the end, we've opted for dropping the count and creating a ticket in Jira to explore option 4 in a follow up. We felt that holding up this AB test anymore is not worth the minor UX improvement the count gets us, but that as we iterate on the auto suggest, this is an important topic to return to. That ticket is here:

https://artsyproduct.atlassian.net/browse/DISCO-896

So that means I'll get a commit in that removes the count and then I think this would be ready to merge.